libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/avoided-emissions.constants.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  MassIdOrganicSubtype,
+  MethodologyBaseline,
+} from '@carrot-fndn/shared/methodologies/bold/types';
+
+export const AVOIDED_EMISSIONS_BASELINE_MATRIX: Record<
+  MassIdOrganicSubtype,
+  Record<MethodologyBaseline, number>
+> = {
+  [MassIdOrganicSubtype.DOMESTIC_SLUDGE]: {
+    [MethodologyBaseline.LANDFILLS_WITH_FLARING_OF_METHANE_GAS]: 0.067,
+    [MethodologyBaseline.LANDFILLS_WITHOUT_FLARING_OF_METHANE_GAS]: 0.227,
+    [MethodologyBaseline.OPEN_AIR_DUMP]: 0.156,
+  },
+  [MassIdOrganicSubtype.EFB_SIMILAR_TO_GARDEN_YARD_AND_PARK_WASTE]: {
+    [MethodologyBaseline.LANDFILLS_WITH_FLARING_OF_METHANE_GAS]: 0.629,
+    [MethodologyBaseline.LANDFILLS_WITHOUT_FLARING_OF_METHANE_GAS]: 1.251,
+    [MethodologyBaseline.OPEN_AIR_DUMP]: 0.975,
```

Comment:
Here is the same rounding difference when using just one ton: 1.251 becomes 1.250 and 0.975 becomes 0..974.